test/selenium/util/TestUser.scala

```diff
@@ -0,0 +1,21 @@
+package selenium.util
+
+import java.time.Duration.ofDays
+
+import com.gu.identity.testing.usernames.TestUsernames
+
+class TestUser {
+
+  private val testUsers = TestUsernames(
+    com.gu.identity.testing.usernames.Encoder.withSecret(Config.testUsersSecret),
+    recency = ofDays(2)
+  )
+
+  private def addTestUserCookies(testUsername: String) = {
+    Driver.addCookie("ANALYTICS_OFF_KEY", "true")
```

Comment:
Where does the name of this cookie come from? I don't think that we are not tracking browser with this Cookie. I would be awesome to add this behaviour to tracker.js